app/services/import/determine-supplementary-billing-flags.service.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+/**
+ * Determines if an imported licence should be flagged for supplementary billing
+ * @module DetermineSupplementaryBillingFlagsService
+ */
+
+const ProcessImportedLicenceService = require('../licences/supplementary/process-imported-licence.service.js')
+const LicenceModel = require('../../models/licence.model.js')
+
+/**
+ * Determines if an imported licence should be flagged for supplementary billing.
+ *
+ * This service is responsible for determining whether a licence imported from NALD should be flagged for
+ * supplementary billing. It compares the licences end dates (such as lapsed, revoked or expired dates) between WRLS
+ * and the transformed data, and if there is a change in the dates allows the licence to go on to determining the flags.
+ *
+ * @param {object} naldLicence - the legacy NALD licence
+ * @param {object} wrlsLicenceId - the WRLS licence data
+ */
+async function go (naldLicence, wrlsLicenceId) {
```

Comment:
We've sat this outside the legacy folder so it theoretically could be used by imports from any system. So, we can't name it `naldLicence`.

I also don't believe there is any confusion as to what system the licence ID relates to.

So, the JSDoc comments and params all need to be updated. Hint, `ProcessImportedLicenceService` might have something you can use!